doc/manuals/admin/cli.md

```diff
@@ -166,6 +166,7 @@ The list of available options is the following:
 -   **-disableCustomNotifications**. Disabled NGSIv2 custom notifications. In particular:
     * `httpCustom` is interpreted as `http`, i.e. all sub-fields except `url` are ignored
     * No `${...}` macro substitution is performed.
+-   **-disableFileLog**. To prevent Orion from logging into a file (default behaviour is use a log file). This option might be useful if you are running on kubernetes.
```

Comment:
An entry in CHANGES_NEXT_RELEASE about the new flag should be added. For instance:

```
- Add: -disableFileLog CLI parameter (ORION_DISABLE_FILE_LOG env var) to prevent Orion from logggin into a file
```

Reply:
Fixed in 521246b5f94f5366050a06501199621e8abd6046